src/components/widgets/search-widget/search-widget-components.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Just a small typo on the component name, it should be `search-widget-component.jsx` in singular (to match the convention of the rest of the components).

Reply:
Done